### PR TITLE
README update for sequelize v6

### PR DESCRIPTION
**Pull request by lukashroch**
Update readme

- how to upgrade to sequelize v6 & sequelize-typescript v2
- outline breaking changes
- sequelize-typescript v1 with v5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,29 +38,96 @@ Decorators and some other features for sequelize (v5).
...
+import { Table, Model } from 'sequelize-typescript';
+
+interface PersonAttributes {
+  id: number;
+  name: string;
+}
+
+interface PersonCreationAttributes {
+  name: string;
+}
```
Comment (intellix) [inline README.md:125]:
```suggestion
interface PersonCreationAttributes extends Omit<PersonAttributes, 'id'> {
}
```

**Review by Divlo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lukashroch) [reply README.md]: Thanks @intellix, added with corresponding import statement. Good idea to keep it consistent with sequelize official TS guide.
...though anyone can freely to construct the interfaces however feels appropriate.

**Review by Divlo — APPROVED** [review]

**Comment by DaPfeifa** [comment]
I hope this can be merged soon.
Otherwise you always run into problems when migrating to Sequelize v6.

**Comment by lukashroch** [comment]
I've squashed the changes and added a note about current state of implementation, so it's more clear what currently works.

- less strict sequelize V6 works quite fine - this is equivalent to what we had so far
- more strict sequelize introduced in v6.2+ is not yet implemented, will need more work

@RobinBuschmann if this could be merged, it would be much appreciated. I should have done the README update along with the code change, so users knew current state and what needs to be done. Thanks!

**Pull request merged**